Tasks/AzureRmWebAppDeployment/operations/PublishProfileUtility.ts

```diff
@@ -0,0 +1,111 @@
+import tl = require('vsts-task-lib/task');
+import { TaskParameters } from './TaskParameters';
+import fs = require('fs');
+import * as Constant from './Constants';
+import path = require('path');
+import Q = require('q');
+
+var packageUtility = require('webdeployment-common/packageUtility.js');
+var parseString = require('xml2js').parseString;
+const ERROR_FILE_NAME = "error.txt";
+
+export class PublishProfileUtility {
+
+    private _publishProfileJs: any = null;
+    private _publishProfilePath: string;
+
+    constructor(publishProfilePath: string) {
+        this._publishProfilePath = publishProfilePath;
+    }
+
+    public async GetTaskParametersFromPublishProfileFile(taskParams: TaskParameters): Promise<any> {
```

Comment:
consider defining a model for the return value

Reply:
done. It was good suggestion.